tests/acceptance/features/apiGraph/deleteUser.feature

```diff
@@ -80,4 +80,3 @@ Feature: delete user
     When the user "Alice" deletes a user "Brian" using the Graph API
     Then the HTTP status code should be "204"
     And user "Brian" should not exist
-    
```

Comment:
You could do `git checkout origin/stable-2.0 tests/acceptance/features/apiGraph/deleteUser.feature` to undo this change